showcase/src/app/demo/pretty-print/demo-pretty-print.component.html

```diff
@@ -0,0 +1,58 @@
+<example-viewer [extensions]="[]"
+				filesPath=""
+				exampleTitle="SHOWCASE.DEMO.PRETTY-PRINT.TRY_IT_YOURSELF">
+	<form>
+		<div class="pretty-print-input mb2">
+			<mat-form-field>
+				<textarea name="unformattedData"
+					   	  [(ngModel)]="unformattedData"
+						  matInput
+						  rows="17"
+					   	  placeholder="{{ 'SHOWCASE.DEMO.PRETTY-PRINT.ENTER_DATA' | translate }}">
+				</textarea>
+			</mat-form-field>
+		</div>
+		<div class="pretty-print-options mb1">
+			<stark-dropdown dropdownId="dataFormatDropdown"
+							[options]="dataFormats"
+							[value]="selectedDataFormat"
+							placeholder="SHOWCASE.DEMO.PRETTY-PRINT.SELECT_DATAFORMAT"
+							(dropdownSelectionChanged)="dataFormatDropdownOnChange($event)">
+			</stark-dropdown>
+			<mat-checkbox class="highlighting-enabled-field ml3"
+						  (change)="$event ? toggleHighlightingEnabled() : null"
+						  [checked]="highlightingEnabled">
+				{{ 'SHOWCASE.DEMO.PRETTY-PRINT.ENABLE_HIGHLIGHTING' | translate }}
+			</mat-checkbox>
+		</div>
+		<div class="pretty-print-result">
+			<stark-pretty-print [data]="unformattedData"
+								[format]="selectedDataFormat"
+								[enableHighlighting]="highlightingEnabled">
+			</stark-pretty-print>
+		</div>
+	</form>
+</example-viewer>
+
+<example-viewer *ngFor="let fileType of fileTypes; trackBy: trackFileTypes"
+				[extensions]="['HTML', 'SCSS', 'TS']"
+				[filesPath]="fileType.path"
+				[exampleTitle]="fileType.title">
+	<mat-tab-group>
+		<mat-tab label="{{ 'SHOWCASE.DEMO.PRETTY-PRINT.ORIGINAL_CODE'  | translate }}">
+			<div class="code-block">{{ fileType.rawData }}</div>
+		</mat-tab>
+		<mat-tab label="{{ 'SHOWCASE.DEMO.PRETTY-PRINT.PRETTY_PRINTED' | translate }}">
```

Comment:
You can achieve the same with a shorter syntax 😉 

`[label]="'SHOWCASE.DEMO.PRETTY-PRINT.PRETTY_PRINTED' | translate"`